react/week1/todolist/src/activities.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import ListItems from "./listitems";
+function HeaderTitle() {
+  return <h1>Welcome to Todo list</h1>;
+}
```

Comment:
Great file structure and while you're at it I'd go one further and give this re-usable HeaderTitle a file for itself but for this assignment does the job!